webview-ui/src/components/chat/chat-view/hooks/useMessageHandlers.ts

```diff
@@ -169,7 +169,14 @@ export function useMessageHandlers(messages: ClineMessage[], chatState: ChatStat
 
 				case "proceed":
 					if (hasContent) {
-						await handleSendMessage(trimmedInput || "", images || [], files || [])
+						await TaskServiceClient.askResponse(
```

Comment:
This seems to be doing the same thing as handleSendMessage?

Reply:
Yeah this might be redundant.